### ISSUE TITLE
Overwriting column_pair_map_expectation

### ISSUE THREAD

**Issue by avanderm** [fivetran/great_expectations#1111]
For the PandasDataset, is it normal behaviour that column pair expectations overwrite each other? I'm using version 0.9.2

Example:
```
import great_expectations as ge
df = ge.read_csv('test.csv')

df.expect_column_pair_values_to_be_equal('id_1', 'id_2')
df.expect_column_pair_values_to_be_equal('id_2', 'id_3')
df.validate()
```
This will end up in only one evaluated expectation, even though the column combinations are distinct.

**Comment by jcampbell** [comment]
Sorry for the delay in this! Unfortunately, this is a real issue with the way that the `column pair map` expectation decorator works; as a workaround, you can add the expectations yourself in the expectation suite JSON.

We're not likely to fix this one yet (but if others are being hit by this, please let me know and we can shift that). I've got plans for a deeper update to the way expectations identify when they are the same, which will be able to solve this much more cleanly.

**Comment by abegong** [comment]
This is still in flight. @jcampbell and I have line of sight to a fix, but it requires some deeper changes that we want to make carefully.

**Comment by jcampbell** [comment]
This is fixed in #1759

**Comment by tdyku** [comment]
Hi!
Is that fixed also for custom expectations? (`@DataAsset.expectation()`)
My custom multicolumn expectations are still overwriting each other and only the last one is saved in the expectation suite.
I am using API v2.
cc: @jcampbell
